.circleci/config.yml

```diff
@@ -1,5 +1,171 @@
-version: 2
+version: 2.1
+
+executors:
+  go:
+    working_directory: ~/go/src/github.com/hashicorp/nomad
+    docker:
+      - image: dantoml/nomad-build-image:e8d69b1e8
```

Comment:
Do we have a DockerHub Organisation that I can use to avoid having this tied to my personal account?

Reply:
We don't currently have this

Reply:
There is a hashicorp DockerHub organization.

Reply:
@cgbaker There is, but engserv wanted us to avoid using that for CI images. There was talk of maybe making `hashicorp-ci` but it hasn't happened yet.

Reply:
We now have hasicorpnomad